src/css/modules/_TileGroup.scss

```diff
@@ -0,0 +1,18 @@
+.coa-TileGroup {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  .coa-Tile {
+    flex-grow: 1 ;
+    flex-shrink: 0;
+    flex-basis: 98%;
+    margin: 1%;
+    @include media($coa-small-screen) {
+      flex-basis: 50% - 4%;
```

Comment:
is this compensating for margin? perhaps 1% become `$tile-margin`. 
Then line 12 becomes: `flex-basis: 50% - (4 * $tile-margin);`.
Is that any more clear?

Reply:
YES will add to next PR